### ISSUE TITLE
How to set cipherpass

### ISSUE THREAD

**Issue by timbrd** [CrunchyData/postgres-operator#1857]
I'd like pgbackrest to encrypt my backups, so I have created a schedule with their corresponding settings:

```bash
$ pgo create schedule my-pg-cluster --schedule="0 2 * * *" --schedule-type=pgbackrest  --pgbackrest-backup-type=full --schedule-opts="--repo1-cipher-type=aes-256-cbc --repo1-cipher-pass=my-secret-cypher-pass" -n my-project-namespace
```

Unfortunately pgbackrest does not allow to specify the password on the command line:

```
[...]
time="2020-09-06T14:35:02Z" level=error msg="command terminated with exit code 127"
time="2020-09-06T14:35:02Z" level=info msg="output=[]"
time="2020-09-06T14:35:02Z" level=info msg="stderr=[bash: option 'repo1-cipher-pass' is not allowed on the command-line\n       HINT: this option could expose secrets in the process list.\n       HINT: specify the option in a configuration file or an environment variable instead.\n]"
time="2020-09-06T14:35:02Z" level=error msg="command terminated with exit code 127"
```

Is there a way to set a password? I guess, I can neither set environment variables nor a configuration file using pgo schedule?

**Comment by jkatz** [comment]
You will need to wait until 4.5.0 to be released (#1767, #1813) as this allows for setting parameters via the `pgbackrest.conf` file.

**Comment by timbrd** [comment]
> You will need to wait until 4.5.0 to be released (#1767, #1813) as this allows for setting parameters via the `pgbackrest.conf` file.

Thanks, good to see it is going to be supported soon.

**Comment by jkatz** [comment]
Glad to hear it. As the change required has already been committed, I'm going to close this one out.

Thanks!
